ssl/src/main/java/io/crate/protocols/ssl/SslContextProvider.java

```diff
@@ -109,7 +95,13 @@ private static void tryUnwrapSslConfigurationException(Throwable e) {
     }
 
     @Override
-    public SslContext get() {
+    public synchronized SslContext get() {
```

Comment:
Synchronized methods can be problematic as they often isolate too much code. I'd use a locking object and then just use a `synchronized(lock)` around the bootstrapping code. The return statement doesn't have to have a lock.